src/services/query.ts

```diff
@@ -58,7 +58,7 @@ const defaultQueryParams = {
 };
 
 export function renderLogQLStreamSelector(filters: AdHocVariableFilter[]) {
```

Comment:
Nit, but this function no longer renders the stream selector, but a sequence of labels.

Reply:
How's `renderLogQLStreamLabels`?

Reply:
I couldn't bring myself to name it renderLogQLLabels, the `LL` hurt me inside

Reply:
Love it. Changed!